R/DBConnection.R

```diff
@@ -106,6 +106,103 @@ setGeneric("dbSendQuery",
   valueClass = "DBIResult"
 )
 
+#' Execute a non-select query
```

Comment:
@hadley I think it best to refer to this sort of thing as executing a statement or command hence supportive of `dbExecute`. You will see my early comment on another issue link to [Wikipedia page on SQL](https://en.wikipedia.org/wiki/SQL#Queries) where SELECT statements are referred to as queries whereas UPDATES etc not.